src/components/CurrencyInputProps.ts

```diff
@@ -10,6 +10,12 @@ export type CurrencyInputProps = Overwrite<
      */
     allowDecimals?: boolean;
 
+    /**
+     * Component to render. Can either be a string e.g. 'input', or a component.
+     */
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Apologies with this line. I'm not 100% sure how to fix the lint errors when the props are specified as `CurrencyInputProps`. My TypeScript is mediocre at best 😂